themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties

```diff
@@ -0,0 +1,224 @@
+doLogIn=Inloggen
+doRegister=Registeer
+doCancel=Annuleer
+doSubmit=Verzenden
+doYes=Ja
+doNo=Nee
+doContinue=Doorgaan
+doAccept=Accepteren
+doDecline=Afwijzen
+doForgotPassword=Wachtwoord vergeten?
+doClickHere=Klik hier
+doImpersonate=Belichamen
```

Comment:
Completely agreed. I considered that translation and many others, but somehow ended up with the creepiest one.